src/assets/translations/sw-SW.json

```
@@ -280,6 +280,9 @@
     "validation.msg.savingsproduct.withdrawalFeeType.is.not.one.of.expected.enumerations": "Aina ya muda wa ada ya kujiondoa ni ya lazima."
   },
   "labels": {
+    "text": {
```

Biome (2.1.2)

[error] The key text was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)